src/main/java/commands/AddCommand.java

```diff
@@ -0,0 +1,49 @@
+package main.java.commands;
+import main.java.tasklist.*;
+import main.java.ui.*;
+import main.java.storage.*;
+import main.java.tasks.*;
```

Comment:
Perhaps you could import each class that you would require explicitly instead of using wildcard imports?